### PR TITLE
Add hover states

### PR DESCRIPTION
**Pull request by tarikhamilton**
Before submitting a pull request, please make sure the following is done:

1. Make sure your updates are reflected in both the React and HTML side of MDL.
2. If you've fixed a bug or added code that should be tested, add tests!
3. Make sure the test suite passes (`npm test`).

### CODE REVIEW

File: components/Button/style.module.scss [hunk 1]
```diff
@@ -24,16 +25,28 @@
   &--default {
     background: $color-snow;
     box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.15);
+
+    &:hover {
```
Comment (josephfusco) [inline components/Button/style.module.scss]: I know we don't have guidelines established for this but we should consider having our focus styles match our hover styles in addition to the blue outline.
```suggestion
    &:hover,
    &:focus {
```

File: components/Button/style.module.scss [hunk 2]
```diff
@@ -24,16 +25,28 @@
...
+      background: darken($color-snow, 10%);
+    }
   }
 
   &--primary {
     background: $color-limerick;
     box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.15);
+
+    &:hover {
```
Comment (josephfusco) [inline components/Button/style.module.scss]:
```suggestion
    &:hover,
    &:focus {
```

File: components/Button/style.module.scss [hunk 3]
```diff
@@ -24,16 +25,28 @@
...
+      background: darken($color-limerick, 5%);
+    }
   }
 
   &--secondary {
     background: $color-deepgrey;
     color: white;
+
+    &:hover {
```
Comment (josephfusco) [inline components/Button/style.module.scss]:
```suggestion
    &:hover,
    &:focus {
```

File: components/Button/style.module.scss (hunk 1 above)
Comment (tarikhamilton) [reply components/Button/style.module.scss]: No guidelines, no problem. Good catch. *Commits suggestion*

**Comment by tarikhamilton** [comment]
@josephfusco says you have requested changes, but I have done them all. Is there any dialogs for you to confirm or anything? I'm lost. If not, just give me a the go ahead and I'll merge.

**Comment by josephfusco** [comment]
> @josephfusco says you have requested changes, but I have done them all. Is there any dialogs for you to confirm or anything? I'm lost. If not, just give me a the go ahead and I'll merge.

Looks like you already resolved those changes I just never hit approve :) All good now

**Pull request merged**